### PR TITLE
Add instagram.com to commonDistractingWebsites

### PR DESCRIPTION
**Pull request by bettinson**
https://www.instagram.com/ is a commonly distracting website now that is fully featured.

### CODE REVIEW

File: Block Management/HostImporter.m [hunk 1]
```diff
@@ -42,6 +42,7 @@ + (NSArray*)commonDistractingWebsites {
 			 @"vine.co",
 			 @"pinterest.com",
 			 @"stumbleupon.com",
+             @"instagram.com",
```
Comment (bettinson) [inline Block Management/HostImporter.m:45]:
```suggestion
			 @"stumbleupon.com",
                          @"instagram.com",
```

Comment (bettinson) [inline Block Management/HostImporter.m:45]:
```suggestion
			 @"stumbleupon.com",
                        @"instagram.com",
```

Comment (bettinson) [inline Block Management/HostImporter.m:45]:
```suggestion
			 @"stumbleupon.com",
                       @"instagram.com",
```

**Comment by cstigler** [comment]
Thanks @bettinson !

**Comment by bettinson** [comment]
:)

**Pull request merged**